documentation.html

```diff
@@ -144,9 +144,68 @@ <h3>Avatar with placeholder initial can be created using the respective avatar-p
                 </iframe>
             </section>
 
+            <section class="component-section">
+                <h1 id="badge-component-heading" class="component-headings">Badge</h1>
+                <h3>Badges are used to highlight an item's status for quick recognition. It is generally used to show the count of notification or status of the user.</h3>
+            
+                <h2 class="component-headings"> Status Badge</h2>
             
+                <h3> Status badge is used to show the status of the user. It has 3 variants online-badge | dnd-badge | idle-badge</h3>
+            
+                <div class="component-preview avatar-component-preview">
+        
+                    <div class="status-badge">
+                        <span class="badge online-badge"></span>
+                    </div>
+
+                    <div class="status-badge">
+                        <span class="badge idle-badge"></span>
+                    </div>
 
+                    <div class="status-badge">
+                        <span class="badge dnd-badge"></span>
+                    </div>
+ 
+                </div>
 
+                <iframe
+                src="https://carbon.now.sh/embed?bg=rgba%2849%2C106%2C181%2C1%29&t=seti&wt=none&l=htmlmixed&width=408&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=36px&ph=39px&ln=false&fl=1&fm=Fira+Code&fs=13px&lh=156%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522status-badge%2522%253E%250A%2509%253Cspan%2520class%253D%2522badge%2520online-badge%2522%253E%253C%252Fspan%253E%250A%253C%252Fdiv%253E%250A%250A%253Cdiv%2520class%253D%2522status-badge%2522%253E%250A%2509%253Cspan%2520class%253D%2522badge%2520idle-badge%2522%253E%253C%252Fspan%253E%250A%253C%252Fdiv%253E%250A%250A%253Cdiv%2520class%253D%2522status-badge%2522%253E%250A%2509%253Cspan%2520class%253D%2522badge%2520dnd-badge%2522%253E%253C%252Fspan%253E%250A%253C%252Fdiv%253E"
+                style="width: 60vw; height: 35vh; border:0; transform: scale(1); overflow:hidden;"
+                sandbox="allow-scripts allow-same-origin">
+                </iframe>
+            
+            </section>
+            
+            <section class="component-section">
+
+                <h2 class="component-headings">Icon badge</h2>
+
+                <h3> Icon badge is used to show the count of the notification or add cart option. It can be created using the class badge-icons icon-badge Font size of badge text is in em. It will change as per the parent element's font-size.</h3>
+                <div class="component-preview avatar-component-preview">
+                    <div class="icon-container">
+                        <img class="icon" src="./assets/house-chimney-solid.svg" alt="">
```

Comment:
The `img` tag should have descriptive `alt` text unless the image is used purely for decoration.